### ISSUE TITLE
Doesn't work with DCH driver

### ISSUE THREAD

**Issue by Chaython** [NateShoffner/Disable-Nvidia-Telemetry#21]
Doesn't work with DCH driver

**Comment by naworm** [comment]
You better don't use DCH driver and this Soft seem's to no longer be maintained

**Comment by Chaython** [comment]
You replied to a 7 month old post... Anyways DCH is being shoved down people's throat.
NV Updater you can silent install without telemetry.
WARNING: NV Updater is not FOSS 
https://www.sys-worx.net/filebase/file/9-nv-updater-nvidia-driver-updater/

**Comment by naworm** [comment]
Yeah it's pretty dirty how DCH is forced. Thanks for the information will check this software out 👍. And the soft do not need dotnet3.5 like this one do, who's a big plus for me.

**Comment by naworm** [comment]
Didn't work properly, waste of time since im using geforce experience

**Comment by Chaython** [comment]
GFE is a giant telemetry and spying tool.
Why do you use it? 
The tool has options to delete or keep GFE etc. So what was the issue?

**Comment by naworm** [comment]
I need ShadowPlay so much ^^
When I delete some other things GFE won't start, tried 3 time with less things, and I do not like how work the tool, you need to reinstall every time.

**Comment by naworm** [comment]
Instead I'm changing this : 
```
HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\NvContainerLocalSystem

"C:\Program Files\NVIDIA Corporation\NvContainer\nvcontainer.exe" -s NvContainerLocalSystem -f "C:\ProgramData\NVIDIA\NvContainerLocalSystem.log" -l 3 -d "C:\Program Files\NVIDIA Corporation\NvContainer\plugins\LocalSystem" -r -p 30000 -st "C:\Program Files\NVIDIA Corporation\NvContainer\NvContainerTelemetryApi.dll"

"C:\Program Files\NVIDIA Corporation\NvContainer\nvcontainer.exe" -s NvContainerLocalSystem -f "C:\ProgramData\NVIDIA\NvContainerLocalSystem.log" -l 3 -d "C:\Program Files\NVIDIA Corporation\NvContainer\plugins\LocalSystem" -r -p 30000
```

**Comment by Chaython** [comment]
Shadowplay isn't any better than constantly recording with OBS. In some ways it's worse...

**Comment by naworm** [comment]
lol yes it’s better for my usage, I post recording my moves in games. I’m using OBS too but for recording or streaming

**Comment by Chaython** [comment]
Yes, well shadowplay is just a constant recording, that caches upto 5 minutes.
So you could just constantly record with OBS. You could also enable replay buffer to "shadowplay" to ram rather than disk. [See the youtube video] You can then also set custom times instead of the fixed 5m of shadowplay.
In some instances Shadowplay wouldn't capture [and it doesn't warn you], it also use to have issues managing the caches, eventually using 1tb of my storage. [Shadowplay highlights still has issues I think, where it caches every highlight, but even though you didn't ask to save highlights it never deleted them and highlights often didn't find things highlight worthy]. 
OBS you have proper control over all this stuff, and can live preview the recording on another display.
https://www.youtube.com/watch?v=oSI_KA4Q8gA

GFE, Shadowplay and DCH are cancerous, unstable, spying, bloat.

**Comment by naworm** [comment]
You're right man, but I won't use OBS for this, I need something easy too configure and use. Will give a try to `Lowkey.gg` and `Medal`.
So I'm re-switching to NV Updater, let's see, Thanks mate ;)

**Comment by katastrophe92** [comment]
> Instead I'm changing this :
> 
> ```
> HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\NvContainerLocalSystem
> 
> "C:\Program Files\NVIDIA Corporation\NvContainer\nvcontainer.exe" -s NvContainerLocalSystem -f "C:\ProgramData\NVIDIA\NvContainerLocalSystem.log" -l 3 -d "C:\Program Files\NVIDIA Corporation\NvContainer\plugins\LocalSystem" -r -p 30000 -st "C:\Program Files\NVIDIA Corporation\NvContainer\NvContainerTelemetryApi.dll"
> 
> "C:\Program Files\NVIDIA Corporation\NvContainer\nvcontainer.exe" -s NvContainerLocalSystem -f "C:\ProgramData\NVIDIA\NvContainerLocalSystem.log" -l 3 -d "C:\Program Files\NVIDIA Corporation\NvContainer\plugins\LocalSystem" -r -p 30000
> ```

Hi. What does this changes?
